src/Label.js

```diff
@@ -1,12 +1,12 @@
 import styled from 'styled-components'
-import { space, fontSize, color, propTypes } from 'styled-system'
+import { color, fontSize, propTypes, space, textAlign } from 'styled-system'
 
 const Label = styled.label`
   font-size: 10px;
```

Comment:
Should move this to a `defaultProp` if you're adding the `fontSize` function

Reply:
Good catch. The original implementation of `Label` does indeed have default prop for `fontSize`, and this `font-size` CSS were already part of the styled component's CSS.

Updated to remove the `font-size` CSS, and only use `fontSize` default prop.